_releases/2023-04-12-1.8.0-released.md

```diff
@@ -0,0 +1,113 @@
+
+Pre-built packages are available on [GitHub Releases](https://github.com/crystal-lang/crystal/releases/tag/1.8.0) and our official distribution channels.
+See [crystal-lang.org/install](https://crystal-lang.org/install/) for installation instructions.
+
+## Stats
+
+In this release we included [XXX changes since the 1.7.3 release](https://github.com/crystal-lang/crystal/pulls?q=is%3Apr+milestone%3A1.7.8) by XX contributors. We thank all the effort put into improving the language! ❤️
+
+Below we list the most remarkable changes in the language, compiler and stdlib.
+
+## Regex: PCRE2
+
+⚠️ A big change in this release is that PCRE2 becomes the default engine for `Regex` ([#12978](https://github.com/crystal-lang/crystal/pull/12978)).
+Support for PCRE2 was added in 1.7 and stabilized in 1.7.3. We feel it's safe to use it by default.
+A fallback to PCRE is still possible, either automatically if PCRE2 is unavailable at compile time, or via the compiler flag `-Duse_pcre`.
+
+> **NOTE:** Package maintainers
+> From 1.8 onwards the Crystal compiler should be built with PCRE2 to ensure consistency.
+
+⚠️ As a measure to ease migration to PCRE2, `Regex::Options` gained more named members which work in both engine versions ([#13223](https://github.com/crystal-lang/crystal/pull/13223)).
+⚠️ Additionally, a new enum `Regex::MatchOptions` was added to hold options specific to match operations ([#13248](https://github.com/crystal-lang/crystal/pull/13248).
+
+⚠️ Unrelated to the upgrade this release includes a bugfix that affects both engine versions.
+Crystal's `String` class is intended to be valid UTF-8 but it's not enforced and
+it's perfectly accepted to contain invalid UTF-8.
+When matching a regex against a string it's necessary to ensure its contents are valid UTF-8 or the libraries may crash.
+This validation mendates a performance overhead and regex matches against strings with invalid UTF-8 bytes will
+now fail for PCRE and older versions of PCRE2. Versions from PCRE2 10.34 onward are capable of dealing
+with invalid UTF-8. Read more in [#13240](https://github.com/crystal-lang/crystal/pull/13240).
+
+> **NOTE:**
+> We intend to add a mechanism for skipping UTF-8 validation if the string is known to be valid.
+> This will be available in the next release.
+
+## LLVM Updates
+
+This release adds support for LLVM 15 ([#13173](https://github.com/crystal-lang/crystal/pull/13173) which includes
+a migration to LLVM's opaque pointers. This has shown to offer a substantial improvement to compilation speed.
+
+> **NOTE:** Package maintainers
+> We recommend building Crystal with LLVM 15 for improved compile performance.
+
+⚠️ On a related note, this release drops support for older LLVM versions. Crystal now requires LLVM 8 or higher ([#12906](https://github.com/crystal-lang/crystal/pull/12906))
+
+LLVM 16 support is in the works ([#13181](https://github.com/crystal-lang/crystal/pull/13181)).
+
+## Misc
```

Comment:
I think platforms require a special place. I'm not sure how to write the Android part, and if we need to expand some of the Windows specific stuff.

```suggestion
## Platform support

This version comes with support for two platforms, and with significant advances in Windows, all listed below. Additionally, we revamped the list of [supported platforms](https://crystal-lang.org/reference/1.8/syntax_and_semantics/platform_support.html) to include specific information of the current level of support.

### AArch64 Android

It is now possible to write headless apps for Android with Crystal.   The details of how to get this working are in the relevant PR ([#13065](https://github.com/crystal-lang/crystal/pull/13065), thanks @HertzDevil)).

### Windows 7

While this platform has reached its EOL, we understand that upgrading might be difficult for some people. Therefore, we accepted a patch to lower the supported Windows version ([#11505](https://github.com/crystal-lang/crystal/pull/11505), thanks @konovod). 

### Windows

This is the complete list of fixes and improvements to Windows incorporated in this release.

- ⚠️ Windows: make `File.delete` remove symlink directories, not `Dir.delete` ([#13224](https://github.com/crystal-lang/crystal/pull/13224), thanks @HertzDevil)
- Leverage `fileapi` for opening files on windows ([#13178](https://github.com/crystal-lang/crystal/pull/13178), thanks @Blacksmoke16)
- Windows: fix error condition when `File.open` fails ([#13235](https://github.com/crystal-lang/crystal/pull/13235), thanks @HertzDevil)
- Improve `File.symlink` on Windows ([#13141](https://github.com/crystal-lang/crystal/pull/13141), thanks @HertzDevil)
- Implement `File.readlink` on Windows ([#13195](https://github.com/crystal-lang/crystal/pull/13195), thanks @HertzDevil)
- Implement `Process.ppid` on Windows ([#13140](https://github.com/crystal-lang/crystal/pull/13140), thanks @HertzDevil)
- OpenSSL: use Windows' system root certificate store ([#13187](https://github.com/crystal-lang/crystal/pull/13187), thanks @HertzDevil)
- ⚠️ Add full stub for Windows signals ([#13131](https://github.com/crystal-lang/crystal/pull/13131), thanks @HertzDevil)

## Misc
```